PhysicsTools/PyTorchAlpaka/test/alpaka/testAlpakaModel.cc

```diff
@@ -0,0 +1,109 @@
+#include <alpaka/alpaka.hpp>
+#include <cppunit/extensions/HelperMacros.h>
+
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/devices.h"
+#include "PhysicsTools/PyTorchAlpaka/interface/GetDevice.h"
+#include "PhysicsTools/PyTorchAlpaka/interface/alpaka/AlpakaModel.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::torchtest {
+
+  constexpr auto modelPath = "PhysicsTools/PyTorchAlpaka/data/linear_dnn.pt";
+
+  // using namespace ALPAKA_ACCELERATOR_NAMESPACE::torch;
```

Comment:
Actually I think it was fine to import the `ALPAKA_ACCELERATOR_NAMESPACE::torch` namespace, as doing that did simplify the code noticeably.